### ISSUE TITLE
Login not working

### ISSUE THREAD

**Issue by aaronbesson** [rubensanroman/react-native-wordpress-api#1]
Hi,

I tried running this, but the login doesn't seem to be working. I am getting "possible unhandled promise rejection (id: 7) TypeError: undefined is not an object. Is there some other configuration other than changing the wp url in the config file? I have wp rest api and jwt authentication plugin installed and active.

thanks!

**Comment by rubensanroman** [comment]
Yes, you need to add a Wordpress admin user and password in the api config file as well

**Comment by rubensanroman** [comment]
Sorry for the poor documentation

**Comment by rubensanroman** [comment]
Are you sure the username and password matches any admin account in Wordpress ?

**Comment by rubensanroman** [comment]
Also, did you set correctly the JWT stuff at Wordpress level ?

**Comment by aaronbesson** [comment]
Hello, thanks for answering, I added the admin the user before as well. I also installed JWT Authentication for WP REST API. What other config is needed on WP side?

**Comment by rubensanroman** [comment]
You need to modify at least 2 files after installation & activation: .htaccess and wp-config.php
